packages/invoices/src/OutboundShipment/DetailView/OutboundServiceLineEdit/useServiceLineColumns.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import {
+  useTranslation,
+  RecordPatch,
+  useColumns,
+  IconButton,
+  XCircleIcon,
+  TextInputCell,
+  CurrencyInputCell,
+  ColumnAlign,
+} from '@openmsupply-client/common';
+import { ServiceItemSearchInput } from '@openmsupply-client/system';
+import { DraftOutboundLine } from './../../../types';
+
+export const useServiceLineColumns = (
+  setter: (patch: RecordPatch<DraftOutboundLine>) => void
+) => {
+  const t = useTranslation('distribution');
+  return useColumns<DraftOutboundLine>([
+    {
+      key: 'serviceItemName',
+      label: 'label.name',
+      width: 350,
+      accessor: ({ rowData }) => rowData?.item?.id,
+      Cell: ({ rowData, column, rows }) => {
+        const id = column.accessor({ rowData, rows }) as string;
+        return (
+          <ServiceItemSearchInput
+            refetchOnMount={false}
+            width={300}
+            onChange={item => item && setter({ ...rowData, item })}
+            currentItemId={id}
+          />
+        );
+      },
+    },
+    {
+      key: 'note',
+      label: 'label.comment',
```

Comment:
comment? 🤷

Reply:
wasn't sure - think it's the best option because it's consistent